README.md

```diff
@@ -1 +1,137 @@
+
+- [🤝 Contributing](#contributing)
+
+- [⭐️ Show your support](#support)
+ 
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+
+# 📖  First Capstone : HTML/CSS & JavaScript project <a name="about-project"></a>
+
+**First Capstone: HTML/CSS & JavaScript** is website for IT meeting project build with HTML, CSS, Javascript , using design guidelines , a mobile first approach, with best practices, Github flow and a descriptive and easy to understand README file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+    <summary>Client</summary>
+        <ul>
+            <li><a  href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+        </ul>
+        <ul>
+            <li><a  href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+        </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+-  Desktop Version
+-  Mobile Version
+-  Home Page
+-  About Page
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://jojo987n.github.io/first-capstone/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web browser.
+- Code editor.
+- Git scm.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command: 
+
+```sh
+ cd my-folder
+ git clone git@github.com:jojo987N/first-capstone.git
+```
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+ cd my-folder
+ npm install
+```
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Deployment
+
+- [Live Demo Link](https://jojo987n.github.io/first-capstone/)
+
+## 👥 Authors <a name="getting-started"></a>
+
+- 👤 Narcisse Jemba
+
+    - GitHub: [@githubhandle](https://github.com/jojo987N)
+    - Twitter: [@JembaNarcisse](https://twitter.com/JembaNarcisse)
+    - LinkedIn: [LinkedIn](https://www.linkedin.com/in/narcisse-jemba-aa9488252/)
+
+- 👤 [Cindy Shin](https://www.behance.net/adagio07)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] add the "More" button on the home page,
+- [ ] implement additional pages, like the tickets page and the schedule page
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just give it a star ⭐️ .
+
+Everybody is welcome to suggest changes in Hello portfolio files.
+
+In order to do it, fork this repository, create a new branch and open a Pull Request from your branch
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] The license link is broken. Please add a license file to make your repo more professional. It will even make it clear what someone can do with your project. 😎️